elements/rh-footer/demo/rh-footer.html

```diff
@@ -332,7 +332,7 @@ <h3 slot="header">About Red Hat</h3>
         </rh-footer-block>
         <rh-footer-block slot="main-secondary">
           <h3 slot="header">Subscribe to our free newsletter, Red Hat Shares</h3>
-          <pfe-cta><a href="#blocks">Sign up now</a></pfe-cta>
+          <rh-cta><a href="#blocks">Sign up now</a></rh-cta>
```

Comment:
Reserving this comment to block auto-merge pending @heyMP's [review](https://github.com/RedHat-UX/red-hat-design-system/pull/349#issuecomment-1217305315).

Reply:
Works perfectly 👍